src/Apps/Order/Components/InsufficientFundsError.tsx

```diff
@@ -4,6 +4,7 @@ import { Spacer, Message, Text } from "@artsy/palette"
 export const InsufficientFundsError: FC = () => {
   return (
     <>
+      <Spacer mt={2} />
```

Comment:
As a general rule, layout spacing should always be defined external to the component; eg, 

```tsx
export const InsufficientFundsError: FC<BoxProps> = (props) => {
  return (
    <Box {...props}>...</Box>
  )
}
```

Then one can do

```tsx
<InsufficientFundsError mt={2} />
```

Or even simpler, where changes don't need to be made to this component: 

```tsx
<Spacer mt={2} /> 

<InsufficientFundsError /> 
```